bpf/tc/attach.go

```diff
@@ -509,14 +502,8 @@ func RemoveQdisc(ifaceName string) error {
 	return libbpf.RemoveQDisc(ifaceName)
 }
 
-func (ap *AttachPoint) ProgramID(tcOpts *libbpf.TCOpts) (string, error) {
-	logCtx := log.WithField("iface", ap.Iface)
-	logCtx.Info("Finding TC program ID")
-	progId, err := libbpf.GetProgID(ap.Iface, string(ap.Hook), tcOpts)
-	if err != nil {
-		return "", errors.New("failed to find TC program")
-	}
-	return strconv.Itoa(progId), nil
+func (ap *AttachPoint) ProgramID() (string, error) {
```

Comment:
Is this no longer used?

Reply:
Yeah. Since we are returning the program ID in Attach. This can be removed. Need to do some cleanup in xdp as well.